index.md

```diff
@@ -2,6 +2,7 @@
 layout: default
 title: Home
 nav_order: 1
+description: GeckoView is library for embedding Gecko in Android apps
```

Comment:
I think this should also be added as the description in the `_config.yml` as I think that sets the metadata for the entire site, not just the index page.